### DISCUSSION TITLE
How to prove academic status in GitHub Teacher Education Application

### DISCUSSION THREAD

**Discussion by GeoGroup** [community/community#52996] [GitHub Education]
Dear Sir/Madam:

I am an associate profesor in Hohai University. All my profile information is updated. But I still cannot prove my self as a lecture. I have try many different ways like the Teacher's card, Employment certificate. All this documents are not able to pass the certification. The email information is as following:
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

I have tried more than 10 times. Would you please give me some tips.

Best Wishes,
Qingxiang Meng

**Comment by samus-aran** [comment]
Hi @GeoGroup 👋 

How did you get on? Did you do anything differently in any later submissions?
